lib/web/mage/sticky.js

```diff
@@ -11,7 +11,26 @@ define([
 
     $.widget('mage.sticky', {
         options: {
-            container: ''
+            container: '',
+            spacingTop: 0,
+            offsetTop: 0,
+            stickyClass: '_sticky'
+        },
+
+        _optionToNumber(option) {
```

Comment:
Hey, thank you @omiroshnichenko, I missed that. ES6 object shorthand is not currently allowed in Magento JavaScript. This should change to:
```js
_optionToNumber: function(option) {
```